UnitsNet.Tests/CustomCode/DensityTests.cs

```diff
@@ -68,5 +70,15 @@ public static void DensityTimesKinematicViscosityEqualsDynamicViscosity()
             DynamicViscosity dynamicViscosity = Density.FromKilogramsPerCubicMeter(2) * KinematicViscosity.FromSquareMetersPerSecond(10);
             Assert.AreEqual(dynamicViscosity, DynamicViscosity.FromNewtonSecondsPerMeterSquared(20));
         }
+
+        private static double SlugsPerCubicFootTolerance { get { return 1e-5; } }
+
+        [Test]
```

Comment:
This test can be removed, it is already covered by the generated `DensityTests.g.cs`.